src/components/Navbar.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import style from '../styles/Navbar.module.css';
+import ghibli from '../assets/ghibli.png';
+
+function Navbar() {
+  return (
+    <nav className={style.navContainer}>
+      <div className={style.logoHolder}>
+        <img src={ghibli} className={style.logo} alt="Ghibli logo" />
+        <h1 className={style.title}>NASA&apos;s Astronomy Picture of the Day</h1>
+      </div>
```

Comment:
- [ OPTIONAL ] great job working on your project so far but I higly recommend making the navbar the same as the given design since it is very important to follow the design guidelines 💯✨ so kindly work on that to make it the same 👍🏻

Reply:
@Reem-lab Thank you for the suggestion! Will do it!